parser/src/test/java/org/dst/parse/Test.java

```diff
@@ -0,0 +1,46 @@
+package org.dst.parse;
+
+import org.dst.parse.po.DstRequestReslut;
+import org.dst.rpc.protobuf.generated.SetProtocol;
+import org.junit.Assert;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.dst.parse.util.CodeUtils.executeExpression;
+
+public class Test {
+
+
+  @org.junit.Test
+  public void testCmdParse() {
+    //put
+    String cmd = "set.put \"k1\" \"v1\" \"v2\" \"v3\"";
+    DstParser dstParser = new DstParser();
+    DstRequestReslut parse = dstParser.parse(cmd);
+    Assert.assertEquals(parse.getRequest().getClass(), SetProtocol.PutRequest.class);
+    //get
+    cmd = "set.get \"k1\"";
+    parse = dstParser.parse(cmd);
+    Assert.assertEquals(parse.getRequest().getClass(), SetProtocol.GetRequest.class);
+    //error grammar
+    try {
+      cmd = "set1.get \"k1\"";
+      parse = dstParser.parse(cmd);
+    } catch (Exception e) {
+      System.out.println(e.toString());
```

Comment:
@senyer 
Could you plz comment out how we should assert an expected exception in unit test?